doc/Developing/Licensing.md

```diff
@@ -1,122 +0,0 @@
-## Contributor Agreement
```

Comment:
I don't think we should remove this file imho

Reply:
Its outdated, and it doesn't add much. we already have the CLA thingy in PRs that covers part of it.
it mostly seem to be a relic from "fork"-epoch, ie mentions observium in every other sentence